src/Files.FullTrust/MessageHandlers/LibrariesHandler.cs

```diff
@@ -48,15 +48,12 @@ private void StartLibrariesWatcher()
             librariesWatcher.EnableRaisingEvents = true;
         }
 
-        public async Task ParseArgumentsAsync(PipeStream connection, Dictionary<string, JsonElement> message, string arguments)
-        {
-            switch (arguments)
+        public Task ParseArgumentsAsync(PipeStream connection, Dictionary<string, object> message, string arguments)
```

Comment:
I think this line may have some merging issue. The message should be `Dictionary<string, JsonElement>`.

Reply:
Oh, I think that was why pipeline builds were failing.